test/import-blacklist_test.go

```diff
@@ -17,10 +17,7 @@ func TestImportsBlacklist(t *testing.T) {
 
 func BenchmarkImportsBlacklist(b *testing.B) {
 	args := []interface{}{"crypto/md5", "crypto/sha1"}
-	var t *testing.T
-	for i := 0; i <= b.N; i++ {
-		testRule(t, "imports-blacklist", &rule.ImportsBlacklistRule{}, &lint.RuleConfig{
-			Arguments: args,
-		})
-	}
+	benchRule(b, "imports-blacklist", &rule.ImportsBlacklistRule{}, &lint.RuleConfig{
```

Comment:
this code is not related with the fix for issue #246, please consider openning other PR

Reply:
Done